akka-docs/src/main/paradox/scala/typed.md

```diff
@@ -3,22 +3,22 @@
 @@@ warning
 
 This module is currently marked as @ref:[may change](common/may-change.md) in the sense
```

Comment:
there should be a java/typed.md also, I wrote that rst page and code examples (so it's only about converting it)

and it should be included in java/index-actors